core/arch/arm/dts/stm32mp15-pinctrl.dtsi

```diff
@@ -1830,356 +1830,356 @@
 
 	uart4_pins_a: uart4-0 {
 		pins1 {
-			pinmux = <STM32_PINMUX('G', 11, AF6)>; /* UART4_TX */
+			pinmux = <STM32_PINMUX_NSEC('G', 11, AF6)>; /* UART4_TX */
 			bias-disable;
 			drive-push-pull;
 			slew-rate = <0>;
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('B', 2, AF8)>; /* UART4_RX */
+			pinmux = <STM32_PINMUX_NSEC('B', 2, AF8)>; /* UART4_RX */
 			bias-disable;
 		};
 	};
 
 	uart4_idle_pins_a: uart4-idle-0 {
 		pins1 {
-			pinmux = <STM32_PINMUX('G', 11, ANALOG)>; /* UART4_TX */
+			pinmux = <STM32_PINMUX_NSEC('G', 11, ANALOG)>; /* UART4_TX */
 		};
 		pins2 {
-			pinmux = <STM32_PINMUX('B', 2, AF8)>; /* UART4_RX */
+			pinmux = <STM32_PINMUX_NSEC('B', 2, AF8)>; /* UART4_RX */
 			bias-disable;
 		};
 	};
 
 	uart4_sleep_pins_a: uart4-sleep-0 {
 		pins {
-			pinmux = <STM32_PINMUX('G', 11, ANALOG)>, /* UART4_TX */
-				 <STM32_PINMUX('B', 2, ANALOG)>; /* UART4_RX */
+			pinmux = <STM32_PINMUX_NSEC('G', 11, ANALOG)>, /* UART4_TX */
+				 <STM32_PINMUX_NSEC('B', 2, ANALOG)>; /* UART4_RX */
```

Comment:
Just a note, we could add att some point the USART1 that is the only one securable

Reply:
I've stated something for that in the commit message:
> On STM32MP15 based devices, UART2/3/4/5/6/7/8 cannot be secured.

Do you think it's not enough?

Reply:
It is